frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1364,6 +1416,27 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 
+	/// Remove the current solutions and proceed to the following round.
+	fn do_force_rotate_round() {
+		<QueuedSolution<T>>::kill();
+		Self::rotate_round();
+	}
+
+	/// Treis to create a snapshot and transitions to the signed phase if successful.
+	fn start_signed_phase() -> Weight {
+		match Self::create_snapshot() {
```

Comment:
should this function assert we are in phase off? Or probably we should try to switch to phase off here?